tests/integration/helpers/open-modal-test.js

```diff
@@ -0,0 +1,134 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { find, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from 'ember';
+import cases from 'qunit-parameterize';
+
+const { RSVP } = Ember;
+
+module('Integration | Helper | open-modal', (hooks) => {
+	setupRenderingTest(hooks);
+
+	test('service modal is called without optional parameters', async function(assert) {
+		let openModalCalled = false;
+
+		const mockedService = Ember.Service.extend({
+			open() {
```

Comment:
You should use [sinon stubs](https://sinonjs.org/releases/latest/stubs/) here. It allows to check for callee arguments easily.